site/docs/master/restic.md

```diff
@@ -283,14 +236,12 @@ should be taken (`backup.velero.io/backup-volumes`)
     - finds the pod volume's subdirectory within the above volume
     - runs `restic backup`
     - updates the status of the custom resource to `Completed` or `Failed`
-1. As each `PodVolumeBackup` finishes, the main Velero process captures its restic snapshot ID and adds it as an annotation
-to the copy of the pod JSON that's stored in the Velero backup. This will be used for restores, as seen in the next section.
+1. As each `PodVolumeBackup` finishes, the main Velero process adds it to the Velero backup in a file named `<backup-name>-podvolumebackups.json.gz`. This will be used for restores, as seen in the next section.
```

Comment:
👍 

Might clarify that this new file gets uploaded to object storage alongside the backup tarball, but it's not part of the tarball itself.